nio/client.py

```diff
@@ -1127,6 +1129,12 @@ def joined_members(self, room_id):
             RequestInfo(RequestType.joined_members, room_id)
         )
 
+    @connected
+    @logged_in
+    def set_displayname(self, displayname):
```

Comment:
Could you please add documentation to this method as well? 

I understand that the HttpClient class isn't really documented but I'm trying to slowly change that.

Reply:
done